packages/v4/src/content/design-guidelines/components/navigation/navigation.md

```diff
@@ -29,27 +29,77 @@ Vertical navigation is hierarchical global navigation that displays navigation o
 * You have secondary navigation items (even if you have less than five primary navigation items)
 * You expect your application to be used on mobile devices
 
-#### Variations
+### Verical navigation variants
 
-**Simple vertical navigation**
+#### Simple vertical navigation
 
 Use a simple, single-level vertical navigation when you only have one level of navigation to display.
 
 <img src="./img/single-level.png" alt="Example of simple vertical navigation" width="634"/>
 
-**Grouped navigation**
+#### Grouped navigation
 
 When you have a small amount of secondary navigation items, you can group your items and display them persistently beneath the primary navigation items.
 
 <img src="./img/two-level.png" alt="Example of grouped vertical navigation" width="634"/>
 
-**Expandable vertical navigation**
+#### Expandable two-level navigation
 
 When you have a large number of secondary navigation items, you can use an expandable navigation to collapse and expand options as needed.
 
 <img src="./img/expandable-nav.png" alt="Example of expandable vertical navigation" width="634"/>
 
+#### Expandable three-level navigation
 
+When you have three levels of navigation items, you can use a three-level expandable navigation to expose tertiary navigation items. Expansion works well when there are small numbers of items at the current level and the entire page hierarchy can be visualized at the same time.
+
+<img src="./img/expandable-nav-tertiary.png" alt="Example of expandable tertiary vertical navigation" width="634"/>
+
+#### Fly-out navigation
+
+Fly-out navigation exposes navigation items nested within a parent node in an overlay panel that appears to the right of the parent item on hover or click. Fly-out menus can be used to expose secondary or tertiary levels of navigation. While fly-out menus can be cascaded, this is not recommended as the mouse interaction required to operate cascading menus can be difficult for some users.
```

Comment:
What do you mean by cascading here? Do you mean when you have more than just the initial level that opens up a set of options (ie: 3 levels)? Or do you mean the way the options are laid out

Reply:
I mean multiple levels of fly-outs, i.e. a fly-out item opening another fly-out. Is there a better term to use?

Reply:
I guess I'm just confused as to what a fly-out would be, if not cascaded? Isn't that the point of a fly-out? Or are you saying that they should be limited to only one "fly-out" if possible?

Reply:
Yes, that's what I'm saying. I'll try to find a better way to word this.